Bouncer/Models/FilterStore/FilterStoreFile.swift

```diff
@@ -83,6 +83,23 @@ extension FilterStoreFile {
         }.eraseToAnyPublisher()
     }
     
+    func update(filter: Filter) -> AnyPublisher<Void, Never> {
+        return Future<Void, Never> { promise in
+            self.fetch()
+                .sink(receiveCompletion: { _ in }, receiveValue: { [weak self] result in
+                    var filters = result
+                    
+                    if let filterIndex = result.firstIndex(where: { $0.id == filter.id }) {
+                        filters[filterIndex] = filter
+                        self?.saveToDisk(filters: filters)
+                        promise(.success(()))
```

Comment:
Properly handling errors here is needed.  In case for example disk is full or there is an error writing to disk, we should handle it gracefully.